code/src/Album.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { AlbumCover } from 'AlbumCover'
+import { AlbumInfo } from 'AlbumInfo'
+import 'Album.css'
+
+export const Album = (props) => (
+    <div className="Album">
+
+    {/* Album cover */}
+    <AlbumCover 
+     item={props.item}/>
```

Comment:
Since the AlbumCover component only needs to get information about the actual image, you can choose to only pass that in as a prop instead of the whole item object.
Something like this:
```
 <AlbumCover image={props.item.images[1].url}/>
```